06-multiple-regression.Rmd

```diff
@@ -260,13 +259,13 @@ get_regression_table(score_model_interaction) %>%
 
 Looking at the regression table output in Table \@ref(tab:regtable-interaction), there are four rows of values in the `estimate` column. While it is not immediately apparent, using these four values we can write out the equations of both lines in Figure \@ref(fig:numxcatxplot1). First, since the word `female` comes alphabetically before `male`, female instructors are the "baseline for comparison" group. Thus, `intercept` is the intercept *for only the female instructors*. 
 
-This holds similarly for `age`. It is the slope for age *for only the female instructors*. Thus, the `r if_else(knitr::is_latex_output(), "darker-colored", "red")` regression line in Figure \@ref(fig:numxcatxplot1) has an intercept of 4.883 and slope for age of -0.018. Remember that for this data, while the intercept has a mathematical interpretation, it has no *practical* interpretation since instructors can't have zero age.
+This holds similarly for `age`. It is the slope for age *for only the female instructors*. Thus, the `r if_else(knitr::is_latex_output(), "darker-colored", "red")` regression line in Figure \@ref(fig:numxcatxplot1) has an intercept of `r get_regression_table(score_model_interaction)$estimate[1]` and slope for age of `r get_regression_table(score_model_interaction)$estimate[2]`. Remember that for this data, while the intercept has a mathematical interpretation, it has no *practical* interpretation since instructors can't have zero age.
 
 What about the intercept and slope for age of the male instructors in the `r if_else(knitr::is_latex_output(), "lighter-colored", "blue")` line in Figure \@ref(fig:numxcatxplot1)? This is where our notion of "offsets" comes into play once again. 
 
-The value for `gendermale` of -0.446 is not the intercept for the male instructors, but rather the *offset*\index{offset} in intercept for male instructors relative to female instructors. The intercept for the male instructors is `intercept + gendermale` = 4.883 + (-0.446) = 4.883 - 0.446 = 4.437. 
+The value for `gendermale` of `r get_regression_table(score_model_interaction)$estimate[3]` is not the intercept for the male instructors, but rather the *offset*\index{offset} in intercept for male instructors relative to female instructors. The intercept for the male instructors is `intercept + gendermale` = `r get_regression_table(score_model_interaction)$estimate[1]` + (`r get_regression_table(score_model_interaction)$estimate[3]`) = `r get_regression_table(score_model_interaction)$estimate[1]` - `r -1*get_regression_table(score_model_interaction)$estimate[3]` = `r get_regression_table(score_model_interaction)$estimate[1] + get_regression_table(score_model_interaction)$estimate[3]`. 
```

Comment:
Same remark here about compute, save, and comment these values in the code chunk titled `regtable-interaction`